### PR TITLE
Token hotfix

### PR DESCRIPTION
**Pull request by frankiebee**
open a new page to https://ethplorer.io/address/${address}

### CODE REVIEW

File: app/scripts/controllers/transactions.js [hunk 1]
```diff
@@ -303,6 +317,7 @@ module.exports = class TransactionController extends EventEmitter {
     // Add the tx hash to the persisted meta-tx object
     const txMeta = this.getTx(txId)
     txMeta.hash = txHash
+    console.log('updating')
```
Comment (Zanibas) [inline app/scripts/controllers/transactions.js]: These would be better as log.debugs

File: ui/app/account-detail.js [hunk 2]
```diff
@@ -255,17 +255,34 @@ AccountDetailScreen.prototype.tabSections = function () {
...
+      //   network,
+      //   tokens,
+      //   addToken: () => this.props.dispatch(actions.showAddTokenPage()),
+      // })
+      return h('.hotFix', {
+        style: {
+          padding: '80px',
+        },
+      }, [`
+        Token List is temporally
```
Comment (Zanibas) [inline ui/app/account-detail.js]: Token lists are temporarily down. You can check you your token balances here.

**Review by 2-am-zzz — CHANGES_REQUESTED** [review]

**Review by 2-am-zzz — DISMISSED** [review]
Please review the comments above.

**Comment by frankiebee** [comment]
screen shot 2017-08-15 at 12 21 26 pm

**Review by 2-am-zzz — APPROVED** [review]

**Pull request merged**